app/src/viewmodels/camera/CameraViewModel.tsx

```diff
@@ -141,5 +159,58 @@ export default function useCameraViewModel(
     takePicture,
     makeChallenge,
     goBack,
+    isInHome,
   };
 }
+
+/**
+ * This function checks if a location is within a group's area.
+ *
+ * @param location the location object
+ * @param group the group object
+ * @returns true if the location is within the group's area, false otherwise
+ */
+function isInGroupArea(location: LocationObject, group: DBGroup) {
+  const distance = getDistance(
+    location.coords.latitude,
+    location.coords.longitude,
+    group.location.latitude,
+    group.location.longitude,
+  );
+  console.log("distance: ", distance);
+  return distance <= group.radius;
+}
```

Comment:
Since we've had so many problems with location, I guess you could add a check in case Location is undefined or null?

Reply:
That function is only used once in a controlled environment. I don't believe it's necessary